.github/workflows/iroha2-dev-pr.yml

```diff
@@ -70,18 +76,23 @@ jobs:
             clang \
             llvm-dev
       - name: Run tests
-        run: cargo test --workspace --no-fail-fast -- --test-threads=1
+        run: cargo test --workspace --no-fail-fast -- --test-threads=4
 
-  test-iroha_network-mock:
+  test-network-mock:
```

Comment:
Now it is more like `test-consensus-with-mocks` as we removed all the network related mocks